src/azure-cli/azure/cli/command_modules/cdn/tests/latest/recordings/test_cdn_custom_domain_https_msft.yaml

```diff
@@ -0,0 +1,2323 @@
+        :\"ImportingUserProvidedCertificate\",\"customHttpsParameters\":{\r\n    \"\
+        certificateSource\":\"AzureKeyVault\",\"certificateSourceParameters\":{\r\n\
+        \      \"@odata.type\":\"#Microsoft.Azure.Cdn.Models.KeyVaultCertificateSourceParameters\"\
+        ,\"subscriptionId\":\"27cafca8-b9a4-4264-b399-45d0c9cca1ab\",\"resourceGroupName\"\
+        :\"clitest.rg000001\",\"vaultName\":\"keyvault000004\",\"secretName\":\"cert000005\"\
+        ,\"secretVersion\":\"35fdb77c063745eeb6dc2fa8404c2cc5\",\"updateRule\":\"\
+        NoAction\",\"deleteRule\":\"NoAction\"\r\n    },\"minimumTlsVersion\":\"TLS12\"\
+        ,\"protocolType\":\"ServerNameIndication\"\r\n  }\r\n}"
+    headers:
+      azure-asyncoperation:
+      - https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/clitest.rg000001/providers/Microsoft.Cdn/operationresults/64fb0c57-3efc-45f1-838c-7c6a2eb85beb?api-version=2019-06-15-preview
+      cache-control:
+      - no-cache
+      content-length:
+      - '847'
+      content-type:
+      - application/json; odata.metadata=minimal
+      date:
+      - Mon, 27 Apr 2020 22:00:31 GMT
+      expires:
+      - '-1'
+      location:
+      - https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/clitest.rg000001/providers/Microsoft.Cdn/operationresults/64fb0c57-3efc-45f1-838c-7c6a2eb85beb/profileresults/profile123/endpointresults/cdn-cli-test-4/customdomainresults/customdomain000003?api-version=2019-06-15-preview
+      odata-version:
+      - '4.0'
+      pragma:
+      - no-cache
+      server:
+      - Microsoft-IIS/8.5
+      strict-transport-security:
+      - max-age=31536000; includeSubDomains
+      x-aspnet-version:
+      - 4.0.30319
+      x-content-type-options:
+      - nosniff
+      x-ms-ratelimit-remaining-subscription-writes:
+      - '1199'
+      x-powered-by:
+      - ASP.NET
+    status:
+      code: 202
+      message: Accepted
+- request:
+    body: null
+    headers:
+      Accept:
+      - application/json
+      Accept-Encoding:
+      - gzip, deflate
+      CommandName:
+      - cdn custom-domain enable-https
+      Connection:
+      - keep-alive
+      ParameterSetName:
+      - -g -n --endpoint-name --profile-name --user-cert-subscription-id --user-cert-group-name
+        --user-cert-vault-name --user-cert-secret-name --user-cert-secret-version
+        --user-cert-protocol-type
+      User-Agent:
+      - python/3.7.7 (Darwin-19.4.0-x86_64-i386-64bit) msrest/0.6.10 msrest_azure/0.6.3
+        azure-mgmt-cdn/4.1.0rc1 Azure-SDK-For-Python AZURECLI/2.4.0
+      accept-language:
+      - en-US
+    method: GET
+    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest.rg000001/providers/Microsoft.Cdn/profiles/profile123/endpoints/cdn-cli-test-4/customDomains/customdomain000003?api-version=2019-06-15-preview
```

Comment:
Do I need to put this at every enable-https request that has a get method? Because some do use the proper uri en method. So I don't know if it really is a mistake anymore. I think the message is getting through right?

Reply:
Same as above. This is a required part of the test. Since these are not unit tests it is impossible to test only the functionality of interest without making addition setup calls and reading state to verify correct behavior. Please let me know if I have not addressed your concerns in my comments.